Gir.toml

```diff
@@ -1334,6 +1334,9 @@ manual_traits = ["GtkMenuExtManual"]
     [[object.function]]
     name = "place_on_monitor"
     version = "3.22"
+    [[object.function]]
+    name = "set_property_attach_widget"
+    ignore = true
```

Comment:
I can't tell `gir` to take a different param, can I? After reading the readme there didn't seem to be a way to do it, but maybe I'm just missing something?

Reply:
Also, did I typo here? gir still generates the function

Reply:
This is property

Reply:
so something like this:
```
    [[object.property]]
    #need be IsA<Widget>
    name = "attach-widget"
    generate = ["get", "notify"]
```

Reply:
It should however actually generate this correctly automatically, like it would already for normal functions :)